source/mongodb/limitations-data-lakes.rst

```diff
@@ -0,0 +1,18 @@
+You can link a `Data Lake
```

Comment:
should the `source/mongodb/limitations-data-lakes.rst` and `source/mongodb/limitations-serverless-instance.rst` files be in the `includes` directory? that's what we usually do w this type of content

Reply:
It _could_ be there but it doesn't have to - we do a mix of both approaches throughout the docs. Here, I chose to keep them under `/mongodb` because they are only shared on two pages that are both in the mongodb section.

The main reason for `/includes` IMO is that we do Giza file generation stuff there (e.g. `steps-foo.yaml` -> `steps/foo.rst`)